src/components/Comparisons/OutfitItem.jsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
```

Comment:
Not sure if this will error out when it tries to export OutfitItem when you haven't defined it? Might be better to at least define it without any data yet or just as a <div> or something for now.

Reply:
defined it!